### DISCUSSION TITLE
Problem with timezone, daylight saving time

### DISCUSSION THREAD

**Discussion by renatodda** [postalserver/postal#2291] [Help with using Postal]
I'm using Postal with GMT-3 Brasilia, but the logs are 1 hour ahead.
I believe it is due to the fact that there is no longer daylight saving time in Brazil and for some reason Postal is identifying daylight saving time and leaving the log with 1 hour more.

**Comment by willpower232** [comment]
Have you just changed the time in the UI for your user or are you running the server in that timezone? It would be worth double checking what the timezone of the server is for comparison.
